docs/machine-requirements.md

```diff
@@ -6,11 +6,10 @@ Whether you want to contribute to Aspire, or just use the latest build of Aspire
 
 On Windows, Visual Studio contains special tooling support for .NET Aspire that you will want to have.
 
-[Visual Studio 2022 version 17.10 Preview](https://visualstudio.microsoft.com/vs/preview/)
+[Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/vs/)
 
-When you install, ensure that both:
+When you install, ensure that:
 * `ASP.NET and web development` Workload is checked.
-* `.NET Aspire SDK` component in `Individual components` is checked.
```

Comment:
@joperezr - do we not have this separate component anymore? Or are we saying it isn't required?